pyproject.toml

```diff
@@ -94,10 +94,14 @@ repository = "https://github.com/mesa/mesa"
 ignore-words-list = ["ist", "hart", "nD"]
 
 [tool.coverage.run]
-source = ["mesa"]
+source_pkgs = ["mesa"]
+relative_files = true
 branch = true
 omit = ["tests/*"]
 
+[tool.coverage.report]
+omit = ["tests/*"]
+
```

Comment:
Best not to skip collecting coverage of the test modules, nor hide it from reports.
```suggestion
```

https://nedbatchelder.com/blog/202008/you_should_include_your_tests_in_coverage.html / https://nedbatchelder.com/blog/201908/dont_omit_tests_from_coverage.html / https://nedbatchelder.com/blog/201106/running_coverage_on_your_tests.html

In fact, it'd be a good idea to include an additional reported check in the Codecov config, requiring 100% code coverage on the test folder: https://github.com/ansible/awx-plugins/blob/274b32a/.codecov.yml#L33-L38.

One might argue that this is out of the scope of this PR and request a follow-up PR but can you prove that all the tests *actually* run in CI w/o that?